package.json

```diff
@@ -1,52 +1,54 @@
 {
-  "name": "jsoning",
-  "version": "0.13.23",
-  "description": "A simple key-value JSON-based persistent lightweight database.",
-  "main": "src/jsoning.js",
-  "types": "src/jsoning.d.ts",
-  "scripts": {
-    "start": "node src/jsoning.js",
-    "test": "nyc ava",
-    "posttest": "npm run delete",
-    "test:npm": "node tests/npmtest.js",
-    "report-coverage": "nyc report --reporter=text-lcov > coverage.lcov && codecov",
-    "docs:build": "node docma.js && bash scripts/createcname.sh",
-    "docs:serve": "cd tests && docma serve",
-    "delete": "rm *.test.json */*.test.json",
-    "lint": "eslint src/jsoning.js",
-    "tsc:types": "tsc src/jsoning.js"
-  },
-  "dependencies": {
-    "write-file-atomic": "^3.0.3"
-  },
-  "devDependencies": {
-    "all-contributors-cli": "^6.20.0",
-    "ava": "^3.15.0",
-    "clean-documentation-theme": "^0.5.2",
-    "codecov": "^3.8.1",
-    "docma": "^3.2.2",
-    "eslint": "^7.21.0",
-    "express": "*",
-    "minami": "^1.2.3",
-    "nyc": "^15.1.0",
-    "prettier": "2.2.1",
-    "typescript": "^4.2.4"
-  },
-  "repository": "khalby786/jsoning",
-  "homepage": "https://jsoning.js.org",
-  "license": "MIT",
-  "keywords": [
-    "json",
-    "database"
-  ],
-  "author": {
-    "name": "Khaleel Gibran",
-    "url": "https://github.com/khalby786/jsoning"
-  },
-  "engines": {
-    "node": ">=0.12"
-  },
-  "files": [
-    "src"
-  ]
+	"name": "jsoning",
+	"version": "0.14.0",
```

Comment:
In light of un-deprecating this package with a new Typescript base, I think we ought to make this v1.0.0

Reply:
also done!